foreman/tests/foreman/management/commands/test_create_missing_processor_jobs.py

```diff
@@ -3,22 +3,16 @@
 from data_refinery_common.models import (
     DownloaderJob,
     DownloaderJobOriginalFileAssociation,
-    Experiment,
-    ExperimentSampleAssociation,
-    Organism,
     OriginalFile,
     OriginalFileSampleAssociation,
     ProcessorJob,
     ProcessorJobOriginalFileAssociation,
     Sample,
-    SurveyJob,
-    SurveyJobKeyValue,
 )
 from data_refinery_foreman.foreman.management.commands.create_missing_processor_jobs import Command
 
 
 class SurveyTestCase(TransactionTestCase):
-    # @patch('data_refinery_foreman.surveyor.external_source.send_job')
```

Comment:
Lets keep this comment until an Issue is created to investigate why it was removed.

Reply:
Done.